crates/derive/src/sources/calldata.rs

```diff
@@ -51,6 +51,7 @@ impl<CP: ChainProvider + Send> CalldataSource<CP> {
                     TxEnvelope::Legacy(tx) => (tx.tx().to(), tx.tx().input()),
                     TxEnvelope::Eip2930(tx) => (tx.tx().to(), tx.tx().input()),
                     TxEnvelope::Eip1559(tx) => (tx.tx().to(), tx.tx().input()),
+                    TxEnvelope::Eip7702(tx) => (tx.tx().to(), tx.tx().input()),
```

Comment:
Super important that we never consider these txs just yet. We just want the ability to observe 7702 txs on L1, but skip over them, without the pipeline crashing due to the tx type being unknown.